Sources/Subscription/Flows/AppStore/AppStoreRestoreFlow.swift

```diff
@@ -34,15 +34,15 @@ public final class AppStoreRestoreFlow {
         case subscriptionExpired(accountDetails: RestoredAccountDetails)
     }
 
-    public static func restoreAccountFromPastPurchase() async -> Result<Void, AppStoreRestoreFlow.Error> {
+    public static func restoreAccountFromPastPurchase(appGroup: String) async -> Result<Void, AppStoreRestoreFlow.Error> {
```

Comment:
@quanganhdo Spotted this discrepancy, the param probably should be `subscriptionAppGroup` for consistency.